imports/ui/ranking/Ranking.js

```diff
@@ -1,72 +1,34 @@
-import { Players } from '../../api/Players.js';
-import { Constants } from '../../api/Constants.js';
+import GameTypeUtils from '../../api/utils/GameTypeUtils';
 
-import './HongKongRanking.html';
-import './JapaneseRanking.html';
+import '../statistics/PlayerModal';
+import './HongKongRanking';
+import './JapaneseRanking';
 import './Ranking.html';
 
-Template.registerHelper('toObj', (args) => {
-		return args.hash;
-});
-
 Template.Ranking.helpers({
 	getInfo(format, player) {
-		let leagueName;
-		let elo;
-		let gamesPlayed;
-		switch (format) {
-		case Constants.GAME_TYPE.JAPANESE:
-			leagueName = player.japaneseLeagueName;
-			elo = player.japaneseElo;
-			gamesPlayed = player.japaneseGamesPlayed;
-			break;
-		case Constants.GAME_TYPE.HONG_KONG:
-			leagueName = player.hongKongLeagueName;
-			elo = player.hongKongElo;
-			gamesPlayed = player.hongKongGamesPlayed;
-			break;
-		default:
-			console.error("Format '" + format + "' is invalid");
-			break;
-		}
-
+		player["elo"] = player["elo"].toFixed(3);
 		return {
-			"leagueName": leagueName,
-			"elo": elo.toFixed(3),
-			"rank": this.rank ? ++this.rank : this.rank = 1,
-			"gamesPlayed": gamesPlayed
+			...player,
```

Comment:
Behold the magic of the [object spread operator](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Spread_operator).